pkg/kubelet/apis/config/validation/validation.go

```diff
@@ -279,5 +279,12 @@ func ValidateKubeletConfiguration(kc *kubeletconfig.KubeletConfiguration, featur
 			fmt.Errorf("invalid configuration: enableSystemLogHandler is required for enableSystemLogQuery"))
 	}
 
+	if kc.ContainerLogMaxWorkers < 1 {
+		allErrors = append(allErrors, fmt.Errorf("invalid configuration: containerLogMaxWorkers must be greater than or equal to 1"))
+	}
+
+	if kc.ContainerLogMonitorInterval.Duration.Seconds() < 3 {
```

Comment:
How did you decide on 3 seconds?

Reply:
@cartermckinnon It was based on this comment https://github.com/kubernetes/kubernetes/pull/114301#discussion_r1483522885